src/Entity/Item.php

```diff
@@ -21,7 +23,7 @@
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
-class Item extends Entity implements StatementListHolder {
+class Item implements Comparable, EntityDocument, FingerprintHolder, StatementListHolder {
```

Comment:
Indeed :) where should Comparable go then? To the end? Honestly this shouldn't matter a lot ;)